integration/kubernetes/init.sh

```diff
@@ -332,7 +332,10 @@ main() {
 		if [ ! -d "${cni_directory}" ]; then
 			sudo mkdir -p "${cni_directory}"
 		fi
-		sudo cp "${crio_repository_path}/contrib/cni/10-crio-bridge.conf" "${cni_directory}"
+		# The config file for cni in cri-o in newer versions has the extension "conflist"
```

Comment:
Instead of saying newer version, can we specify the exact version i.e. `1.26`

Reply:
Thanks. Fixed